src/Sarif/Core/StackFrame.cs

```diff
@@ -55,10 +55,7 @@ public static StackFrame Create(System.Diagnostics.StackFrame dotNetStackFrame)
 
             if (nativeOffset != -1)
             {
-                stackFrame.Properties = new Dictionary<string, string>
-                {
-                    { "NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture) }
-                };
+                stackFrame.SetProperty("NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture));
```

Comment:
Coming next, yes?

---
In reply to: [62750080](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62750080) [](ancestors = 62750080)